__tests__/app.test.js

```diff
@@ -76,6 +75,11 @@ describe('GET /api/articles/:article_id', () => {
         })
     });
     test('should return 404 Not Found status when request to a non-existent article', () => {
-
+        return request(app)
```

Comment:
Would be good to have a test that checks what happens when we try to access an invalid article e.g /api/articles/bananas